syncd/ComparisonLogic.cpp

```diff
@@ -2806,6 +2806,8 @@ void ComparisonLogic::cretePreMatchForLagMembers(
     }
 }
 
+#define PFC_ACL_RULE_PRIORITY "999"
```

Comment:
@neethajohn can you confirm if hardcoding 999 as a PFC ACL priority will work across branches?

Reply:
no clue, this is hardcoded as well in OA